packages/vue-script-setup-converter/src/lib/convertSrc.ts

```diff
@@ -89,12 +89,12 @@ export const convertSrc = (input: string) => {
       })
   );
 
+  statements.addStatements(components);
```

Comment:
[The insertion of defineAsyncComponent should be done before definePag…](https://github.com/wattanx/wattanx-converter/pull/60/commits/13bb4fb8e89001d3581d24155e953265356f91bd)